README.md

```diff
@@ -176,6 +177,18 @@ http server framework. Some new events were added related to fastify setup.
     assembled via all the previous events has been added to `fastify` so
     that it can be served on `bedrock-server.ready`.
 
+# License
+
+Bedrock and all Bedrock modules are:
+
+    Copyright (c) 2011-2024 Digital Bazaar, Inc.
```

Comment:
Git history for this module starts in 2013.  Where did 2011 come from?  I this this should be correct per-module and not copied for all Bedrock modules.
```suggestion
    Copyright (c) 2013-2024 Digital Bazaar, Inc.
```

Reply:
The plan in https://github.com/digitalbazaar/bedrock/pull/117 is to remove this copyright line for the README as it's confusing and a pain to maintain.

Reply:
Reminder that this (and other changes?) need to be synced over.